scripts/actions/spells/trust/zeid_ii.lua

```diff
@@ -14,6 +14,10 @@ end
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
 
+    xi.trust.teamworkMessage(mob, {
+        [xi.magic.spell.LION_II] = xi.trust.messageOffset.TEAMWORK_1,
```

Comment:
Huh, I would have thought he had more than just Lion. Get someone to check if he also does a modified line for LION_I and not just LION_II.

What do his message dumps say?

Reply:
(My local builds are playing up at the moment, I can't dump them for you like I did for Lion I and II)

Reply:
xi.trust.teamworkMessage replaces xi.trust.message(, otherwise you're going to get 2x spawn messages.